examples/jobs/dp-bert-launch-job.md

```diff
@@ -50,12 +54,23 @@ sbatch  --exclusive --nodes=16 --wrap "srun ./run_dp_bert_large_hf_pretrain_bf16
 
 Again, the job id will be displayed by sbatch and you can follow the training by inspecting the file `slurm_<job id>.out` file generated in `~/examples/dp_bert_hf_pretrain`.
 
+### Cluster scalability
+
+There are two measures of scalability of a cluster: strong scaling and weak scaling. Typically, for model training, the need is to speed up training run, because usage cost is determined by sample throughput for rounds of gradient updates. In such scenario, we need to adjust gradient accumulation micro-step according to number of compute nodes. This is achieved with the following in the downloaded training shell script `run_dp_bert_large_hf_pretrain_bf16_s128.sh`:
```

Comment:
should we elaborate on the strong and weak scaling and then detail gradient accumulation ?

Reply:
Fixed. See new commit: https://github.com/aws-neuron/aws-neuron-parallelcluster-samples/pull/17/commits/6456e8eeec7c015832c27d3f05e3d197ca642249